gui/functionality/idact_notebook.py

```diff
@@ -114,6 +114,8 @@ def deploy_notebook(self):
                 notebook.open_in_browser()
                 sleep_until_allocation_ends(nodes=nodes)
         except:  # noqa, pylint: disable=broad-except
+            self.window = ErrorApp()
```

Comment:
Maybe `e` has some sort of a title that can be passed as an error message?

Reply:
I changed message to "An error occured while deploing notebook".